app/views/groups/_group.html.erb

```diff
@@ -0,0 +1,8 @@
+<li itemscope itemtype="http://schema.org/Code">
```

Comment:
This is for a list of groups that a user will see when they click the "Groups" button on their profile page.  This lists all groups the user is a member of.